apps/desktop/scripts/verify-testflight-build.sh

```
@@ -0,0 +1,177 @@
+  LAUNCH_OK="pass"
+  for _ in $(seq 1 15); do
+    sleep 1
+    if ! pgrep -f "$APP_PATH/Contents/MacOS/" >/dev/null 2>&1; then
+      LAUNCH_OK="fail"   # died after starting — that IS the fossil failure
+      break
+    fi
+  done
+  check "App still running 15s after launch (did not crash)" "$LAUNCH_OK"
+fi
+
+CRASHES_AFTER=$(ls -1 "$CRASH_DIR" 2>/dev/null | grep -c -i "drafto" || true)
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)